elpy/jedibackend.py

```diff
@@ -179,7 +179,7 @@ def rpc_get_oneline_docstring(self, filename, source, offset):
                 raw_name = definition.name
                 name = '{} {}'.format(raw_name, definition.type)
                 doc = definition.docstring().split('\n')
-            elif definition.type in ['instance']:
+            elif definition.type in ['instance'] and assignment not None:
```

Comment:
Wait, isn't there supposed to be an `is` in the statement as well?
```suggestion
            elif definition.type in ['instance'] and assignment is not None:
```

Reply:
This still triggers an error. 
A quick fix as below worked, although I am not sure it it fits in the context of this function. 
            elif definition.type in ['instance'] and (assignment is not None) and hasattr(assignment, "name"):

Reply:
Sorry for the delay on my side.

Thanks for the fixes.
In my understanding, assignments should always have a `name` attribute.
But better safe than sorry.